pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -444,6 +446,23 @@ type AlertmanagerWebSpec struct {
 	Timeout *uint32 `json:"timeout,omitempty"`
 }
 
+// AlertmanagerLimitsSpec defines the limits command line flags when starting Alertmanager.
+// +k8s:openapi-gen=true
+type AlertmanagerLimitsSpec struct {
+	// The maximum number active and pending silences. This corresponds to the
+	// Alertmanager's `--silences.max-silences` flag.
+	// It requires Alertmanager >= v0.28.0.
+	//
+	// +optional
+	MaxSilences *uint32 `json:"maxSilences,omitempty"`
```

Comment:
to follow the Kubernetes API conventions (`All public integer fields MUST use the Go int32 or Go int64 types [...] prefer int32 to int64`)
```suggestion
        // +kubebuilder:validation:Minimum:=0
	MaxSilences *int32 `json:"maxSilences,omitempty"`
```

Reply:
Thanks @simonpasquier 
Got it
Fixed